test/valgrind-suppression/extract_suppressions.py

```diff
@@ -0,0 +1,64 @@
+import argparse
+import re
+
+parser = argparse.ArgumentParser()
+parser.add_argument("input_file", help="path to input file")
+parser.add_argument("output_file", help="path to output file")
+args = parser.parse_args()
+
+input_file = args.input_file
+output_file = args.output_file
+
+unique_blocks = set()
+current_block = []
+inside_block = False
+total_blocks = 0
+
+# Matches the time stamp
+ts_re = re.compile(r"^\d{4}-\d{2}-\d{2}T\S+\s+(.*)$")
+
+with open(input_file) as f:
+    for raw_line in f:
+        line = raw_line.rstrip()
+
+        # Strip timestamp prefix if present
+        m = ts_re.match(line)
```

Comment:
```suggestion
        ts_match = ts_re.match(line)
```
Rename `m` to `ts_match`?